docs/userGuide/plugins/web3Form.md

```diff
@@ -0,0 +1,108 @@
+### Plugin: Web3Forms 
+
+This plugin allows you to create forms whose response will be sent directly to your email, using the [Web3Form](https://web3forms.com/) API. 
+<box type="warning" seamless>
+Emails from Web3Forms may end up in spam/junk mail.
```

Comment:
Thanks for adding this in! Could we rephrase this as an actionable tip for users? e.g. "Don't forget to check your spam/junk folder if your form responses aren't appearing in your inbox!" 
We could also use a `tip` type for the `<box>` to reduce the number of consecutive warning messages :)